.gitlint

```diff
@@ -0,0 +1,2 @@
+--subject-regex=#\d+ - .+
```

Comment:
@g4s8 it's quite dangerous to ask people to start their commit with the `#` character because it is the symbol for a comment and so it means it's going to be impossible to use a text editor to edit the comment (it will be considered as a comment and the line won't be taken by git). That's why cactoos wrapped it into parenthesis I think.